js_modules/dagit/packages/core/src/runs/RunTableNew.tsx

```diff
@@ -291,25 +325,55 @@ const RunRow: React.FC<{
               </Link>
             </Box>
           )}
-          <RunTags
-            tags={
-              [
-                run.pipelineSnapshotId
-                  ? {
-                      key: DagsterTag.SnapshotID,
-                      value: run.pipelineSnapshotId?.slice(0, 8) || '',
-                      link: run.pipelineSnapshotId
-                        ? getPipelineSnapshotLink(run.pipelineName, run.pipelineSnapshotId)
-                        : undefined,
-                    }
-                  : null,
-                run.tags.find((tag) => tag.key === DagsterTag.Partition),
-              ].filter((x) => x) as TagType[]
-            }
-            mode={isJob ? (run.mode !== 'default' ? run.mode : null) : run.mode}
-            onAddTag={onAddTag}
-          />
+          <Box flex={{direction: 'row', gap: 8, wrap: 'wrap'}}>
+            <Box>
+              <BaseTag
+                fillColor={Colors.Gray100}
```

Comment:
Is this very different from a regular `Tag` with default intent?

Reply:
I wanted to stick to the figma colors

Reply:
can we use Josh's latest colors or should we find what's closest? it changes the aesthetic a bit so I tend to stick to the figma colors